packages/dd-trace/src/appsec/iast/iast-log.js

```diff
@@ -0,0 +1,117 @@
+'use strict'
+
+const log = require('../../log')
+const telemetryLogs = require('./telemetry/logs')
+const { calculateDDBasePath } = require('../../util')
+
+const ddBasePath = calculateDDBasePath(__dirname)
+const EOL = '\n'
+const STACK_FRAME_LINE_REGEX = /^\s*at\s/gm
+
+function sanitize (logEntry, stack) {
+  if (!stack) return logEntry
+
+  let stackLines = stack.split(EOL)
+
+  let firstIndex = -1
+  for (let i = 0; i < stackLines.length; i++) {
+    if (stackLines[i].match(STACK_FRAME_LINE_REGEX)) {
+      firstIndex = i
+      break
+    }
+  }
+
+  const isDDCode = firstIndex > -1 ? stackLines[firstIndex].includes(ddBasePath) : false
+  stackLines = stackLines
+    .filter((line, index) => (isDDCode && index < firstIndex) || line.includes(ddBasePath))
+    .map(line => line.replace(ddBasePath, '/'))
```

Comment:
Should we use `path.sep` instead of `/`?

Reply:
yes